site/sfguides/src/hex_titanic/titanic.md

```diff
@@ -0,0 +1,192 @@
+author: aefendic@hex.tech and chase.romano@snowflake.com
+id: end_to_end_model_development_with_snowpark_and_hex
+summary: This lab will walk you through how to use Snowflake and Hex.
+categories: data-science-&-ml,partner-integrations
+environments: web
+status: Published
+feedback link: https://github.com/Snowflake-Labs/sfguides/issues
+tags: Hex, Notebooks, Partner Connect
+
+# End to End Model Development with Hex and Snowpark
+
+<!-- ------------------------ -->
+## Lab Overview 
+Duration: 5
+
+In this quickstart, we will be utilizing the classic Titanic dataset to determine if we would survive the infamous maiden voyage. Highlights of the quickstart include, preprocessing, model training, registry, and finally an interactive Hex app for inference. By the end of this lab you will learn how to leverage Snowpark ML, Hex's Snowpark intergration, and model Registry in Snowflake.
+
+### Prerequisites
+- Familiarity with basic Python and SQL 
+- Familiarity with training ML models
+- Familiarity with data science notebooks
+
+
+### What You'll Learn
+* How ot perform preprocessing with Snowpark ML
+* How to perform model training with Snowpark ML
+* How to register the model in Snowflakes Registry
+* How models are "deployed" in Snowflake
+* How to perform inference using Snowpark and SQL against the registered model
+
+
+### If You Would Like to Create All Assets In The Lab You’ll Need:
+- A [Snowflake](https://signup.Snowflake.com/) Account (if you are using an enterprise account through your organization, it is unlikely that you will have the privileges to use the `ACCOUNTADMIN` role, which is required for this lab).
+
+- A Hex account which will be automatically created via Partner Connect. Please follow the steps below.
+
+<!-- ------------------------ -->
+## Creating a Snowflake account
```

Comment:
Remove this step and make it part of the "Prerequisites" section under "Overview" step.

See https://quickstarts.snowflake.com/guide/getting_started_with_dataengineering_ml_using_snowpark_python/index.html